include/blobify/endian.hpp

```diff
@@ -5,8 +5,12 @@
 // C++20 provides std::endian
 #include <bit>
 #elif !defined(_WIN32)
+#ifdef __APPLE__
```

Comment:
Mind folding this ifdef into the outer one? I.e.

```cpp
#if defined(__cpp_lib_endian)
...
#elif defined(__APPLE__)
...
#elif !defined(_WIN32)
...
#endif
```

Reply:
Fixed